sklearn/metrics/tests/test_pairwise.py

```diff
@@ -4,8 +4,12 @@
 from numpy import linalg
 
 from scipy.sparse import dok_matrix, csr_matrix, issparse
-from scipy.spatial.distance import cosine, cityblock, minkowski, wminkowski
+from scipy.spatial.distance import cosine, cityblock, minkowski
 from scipy.spatial.distance import cdist, pdist, squareform
+# wminkowki metric will be deprecated starting from scipy 1.6
+# and removed in scipy 1.8
+from scipy import version
+from scipy.spatial.distance import wminkowski
```

Comment:
This import should probably be protected under because we will probably get an ImportError in the future (not 1.6 but in 1.8 or similar).

```python
try:
    from scipy.spatial.distance import wminkowski
except Import Error:
    # In scipy 1.6.0, wminkowski is deprecated and minkowski
    # should be used instead.
    from scipy.spatial.distance import minkowski as wminkowski
```